### PR TITLE
Make it work (ish) on Windows 7

### PR DESCRIPTION
**Pull request by CreepySkeleton**

### CODE REVIEW

**Comment by CreepySkeleton** [comment]
LOL

File: src/term.rs [hunk 1]
```diff
@@ -403,21 +403,22 @@ impl Term {
...
 
-    #[cfg(all(windows, features = "windows-console-colors"))]
+    #[cfg(all(windows, feature = "windows-console-colors"))]
     fn write_through(&self, bytes: &[u8]) -> io::Result<()> {
         if self.is_msys_tty {
             self.write_through_common(bytes)
         } else {
             use winapi_util::console::Console;
 
+
```
Comment (pksunkara) [inline src/term.rs:413]: Unneeded line.

File: src/windows_term.rs [hunk 2]
```diff
@@ -382,7 +382,7 @@ pub fn set_title<T: Display>(title: T) {
     }
 }
 
-#[cfg(feature = "windows-console-colors")]
+#[cfg(all(windows, feature = "windows-console-colors"))]
```
Comment (pksunkara) [inline src/windows_term.rs:385]: You don't need this change since this file exists only in `windows`

Comment (CreepySkeleton) [reply src/windows_term.rs:385]: But it makes it explicit

**Comment by CreepySkeleton** [comment]
Any idea about Invalid Handle?

**Comment by pksunkara** [comment]
That's not related to this PR.

**Comment by pksunkara** [comment]
Oops, looks like it is related to this PR, because of the `?` you added. The doctest isn't working because of that.

**Comment by CreepySkeleton** [comment]
Well, if `Console::stdout/stderr` has returned an error, there's something very wrong with the environment already.

**Comment by pksunkara** [comment]
Yeah, we can't rely on Console and will have to do the logic ourselves

**Pull request merged**